alphastats/gui/utils/analysis_helper.py

```diff
@@ -79,26 +85,23 @@ def download_preprocessing_info(plot):
     )
 
 
-def read_uploaded_file_into_df(file, decimal="."):
-    filename = file.name
+def _read_file_to_df(file: UploadedFile, decimal: str = ".") -> Optional[pd.DataFrame]:
```

Comment:
Move this to ui_helper, too if it is used by several pages

Reply:
done in https://github.com/MannLabs/alphapeptstats/pull/307